src/components/ProjetoCard.tsx

```
@@ -43,43 +43,51 @@ export function ProjetoCard({
-                    <p key={i} className="text-sm text-gray-700">
-                      "{p.nome}"
-                    </p>
-                  ))
-                ) : (
-                  <p className="text-sm text-gray-500 italic">
-                    Nenhum responsável definido
-                  </p>
-                )}
-              </div>
-            </div>
+              {(projeto.equipeExecutora && projeto.equipeExecutora.length > 0 ||
```

ESLint

[error] Replace `··{(projeto.equipeExecutora·&&·projeto.equipeExecutora.length·>·0` with `{((projeto.equipeExecutora·&&·projeto.equipeExecutora.length·>·0)`

(prettier/prettier)